packages/cli/src/commands/env/ls.ts

```diff
@@ -71,14 +74,32 @@ export default async function ls(
         project.name
       )} ${chalk.gray(lsStamp())}`
     );
-    // eslint-disable-next-line no-console
-    console.log(getTable(envs));
+    client.stdout.write(`${getTable(envs, customEnvs)}\n`);
```

Comment:
I went from `console.log` -> `client.stdout.write` to remove the eslint disable, and it shouldn't change behavior. (Side note: maybe we should change all eslint disabled `console.log`s to `client.stdout.write`?) There's actually an existing integration test that expects this to be `stdout` too.

Good question about whether this was actually desired behavior though. Looking into [clig.dev](https://clig.dev/#the-basics), sounds like `stdout` makes sense since it's the primary output of the command:

| Send output to stdout. The primary output for your command should go to stdout.